client/components/masterbar-styled/default-contact.tsx

```diff
@@ -0,0 +1,83 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
```

Comment:
This is a copy of https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/checkout-thank-you/marketplace/masterbar-styled/default-contact.tsx